Playground/index-local.html

```diff
@@ -59,8 +59,9 @@
             BABYLONDEVTOOLS.Loader       
                 .require("index.js")
                 .load(() => {
+                    BABYLON.GLTF2.Loader.Extensions.EXT_meshopt_compression.DecoderPath = "../dist/preview%20release/meshopt_decoder.module.js";

```

Comment:
Huh.. Looks like we are missing some code here for the local PG.

```javascript
                    BABYLON.DracoCompression.Configuration.decoder = {
                        wasmUrl: "../../dist/preview%20release/draco_wasm_wrapper_gltf.js",
                        wasmBinaryUrl: "../../dist/preview%20release/draco_decoder_gltf.wasm",
                        fallbackUrl: "../../dist/preview%20release/draco_decoder_gltf.js"
                    };
                    BABYLON.GLTFValidation.Configuration = {
                        url: "../../dist/preview%20release/gltf_validator.js"
                    };
```

If you search for `dracoCompression.Configuration` in the code base, it would be good to add your DecoderPath to all the various places that should have it.

Reply:
Yup - will do - I stole this from Sandbox, but I was using Playground for testing.